velox/exec/tests/utils/PlanBuilder.cpp

```diff
@@ -1014,6 +1015,287 @@ PlanBuilder& PlanBuilder::unnest(
   return *this;
 }
 
+namespace {
+std::string toString(const std::vector<FunctionSignaturePtr>& signatures) {
+  std::stringstream out;
+  for (auto i = 0; i < signatures.size(); ++i) {
+    if (i > 0) {
+      out << ", ";
+    }
+    out << signatures[i]->toString();
```

Comment:
nti: Something like this should also do the same thing : fmt::format("{}",fmt::join(signatures,","));